### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by mouhajer-ch**
### JIRA link

Related to [ROE-82](https://companieshouse.atlassian.net/browse/ROE-82)

### Change description

Add PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+### Change description
+
+
+
+### Work checklist
+
+- [ ] Tests added where applicable
+- [ ] UI changes look good on mobile
```
Comment (mwestacott) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: Don't think we need the mobile check.

Comment (mouhajer-ch) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: I will remove it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] UI changes meet accessibility criteria
```
[the remark this answers is not in the document]
Comment (mouhajer-ch) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: Just following GDS styling https://design-system.service.gov.uk/?_ga=2.20058104.1721178051.1647845419-830291819.1619359936 but for more info I will post a good link

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+### Merge instructions
+
+We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:
```
[the remark this answers is not in the document]
Comment (mouhajer-ch) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Sorry, I do not know where is the "this, this"

[the remark this answers is not in the document]
Comment (SVarallo) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: As in "To achieve this, this commit should be structured as follows:"

[the remark this answers is not in the document]
Comment (mouhajer-ch) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Ok, sorry I did not get it, I will update soon

**Review by mwestacott — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Pull request merged**